metricbeat/docs/running-on-docker.asciidoc

```diff
@@ -19,6 +19,7 @@ docker run \
   --mount type=bind,source=/sys/fs/cgroup,target=/hostfs/sys/fs/cgroup,readonly \ <2>
   --mount type=bind,source=/,target=/hostfs,readonly \ <3>
   --net=host \ <4>
+  --user root --privileged \ <5>
```

Comment:
Agree, I will change it. Root user is still needed because of https://github.com/moby/moby/issues/38664
As an alternative we could set these capabilities in the extended attributes as we do in heartbeat and auditbeat, but I think that these capabilities are too powerful to be enabled by default.